tools/build/Makefile.targets

```diff
@@ -234,3 +234,7 @@ check-bindings-nodejs: bindings-nodejs
 PHONY += check-bindings-nodejs
 
 endif #neq($(USE_NODEJS),)
+
+gen-aliases:
+	$(Q)$(MKDIR) -p $(build_aliasesdir)
+	$(FLOW_NODE_TYPE_ALIAS) --file-name=default --node-description-dir=$(flow-dir) --output-dir=$(build_aliasesdir)
```

Comment:
not a good interface, get the output filename (including number + `.json`) then all the files you want to use for that, not a directory. This way to do an alias file for 2 files you'd need to create a directory for it.